src/components/Explore/TokenRow.tsx

```diff
@@ -207,6 +236,39 @@ function HeaderCell({
   return <Trans>{getHeaderDisplay(category, timeframe)}</Trans>
 }
 
+export function LoadingRow({ key }: { key: string }) {
+  return (
```

Comment:
do you think we should do this inline or make a separate row? my opinion is do it inline in the main row

Reply:
another option would be to make a `Row` component and then have `LoadedRow` and `LoadingRow` that use it. then the logic isn't coupled but also is abstracted

Reply:
tokenRow, LoadedRow, and LoadingRow